HW_23/custom_word_generator.py

```diff
@@ -0,0 +1,29 @@
+import random
+
+import nltk
+from nltk.corpus import words
+
+nltk.download('words')
+
+
+def custom_word_generator(num: int) -> str:
+    if num > 10_000:
+        raise ValueError("The number of words should not exceed 10 000.")
+    word_list = list(
+        set(words.words()))  # Remove duplicate words from the sequence to make sure that function
+    # random.sample() return only unique words.
```

Comment:
remove it, I dont care about available unique words. We need uniq words in this range))